### ISSUE TITLE
Crash on Startup - Windows 10

### ISSUE THREAD

**Issue by IAmDragon** [topher-au/WoWmapper#4]
Hey there, crash on startup with Windows 10 using the latest version at the time of this post. Crashes when I start it, exit it, and when I click the plugins button. Any ideas?

See the end of this message for details on invoking 
just-in-time (JIT) debugging instead of this dialog box.

************** Exception Text **************
System.NullReferenceException: Object reference not set to an instance of an object.
   at WoWmapper.MainForm.ExitApp()
   at System.Windows.Forms.Form.OnFormClosing(FormClosingEventArgs e)
   at System.Windows.Forms.Form.WmClose(Message& m)
   at WoWmapper.MainForm.WndProc(Message& m)
   at System.Windows.Forms.NativeWindow.Callback(IntPtr hWnd, Int32 msg, IntPtr wparam, IntPtr lparam)

************** Loaded Assemblies **************
mscorlib
    Assembly Version: 4.0.0.0
    Win32 Version: 4.6.1063.1 built by: NETFXREL3STAGE
    CodeBase: file:///C:/Windows/Microsoft.NET/Framework64/v4.0.30319/mscorlib.dll
----------------------------------------
WoWmapper
    Assembly Version: 0.4.1.0
    Win32 Version: 0.4.1.0
    CodeBase: file:///C:/Program%20Files%20(x86)/WoWmapper/WoWmapper.exe
----------------------------------------
System.Windows.Forms
    Assembly Version: 4.0.0.0
    Win32 Version: 4.6.1038.0 built by: NETFXREL2
    CodeBase: file:///C:/WINDOWS/Microsoft.Net/assembly/GAC_MSIL/System.Windows.Forms/v4.0_4.0.0.0__b77a5c561934e089/System.Windows.Forms.dll
----------------------------------------
System
    Assembly Version: 4.0.0.0
    Win32 Version: 4.6.1038.0 built by: NETFXREL2
    CodeBase: file:///C:/WINDOWS/Microsoft.Net/assembly/GAC_MSIL/System/v4.0_4.0.0.0__b77a5c561934e089/System.dll
----------------------------------------
System.Drawing
    Assembly Version: 4.0.0.0
    Win32 Version: 4.6.1038.0 built by: NETFXREL2
    CodeBase: file:///C:/WINDOWS/Microsoft.Net/assembly/GAC_MSIL/System.Drawing/v4.0_4.0.0.0__b03f5f7f11d50a3a/System.Drawing.dll
----------------------------------------
System.Configuration
    Assembly Version: 4.0.0.0
    Win32 Version: 4.6.1038.0 built by: NETFXREL2
    CodeBase: file:///C:/WINDOWS/Microsoft.Net/assembly/GAC_MSIL/System.Configuration/v4.0_4.0.0.0__b03f5f7f11d50a3a/System.Configuration.dll
----------------------------------------
System.Core
    Assembly Version: 4.0.0.0
    Win32 Version: 4.6.1038.0 built by: NETFXREL2
    CodeBase: file:///C:/WINDOWS/Microsoft.Net/assembly/GAC_MSIL/System.Core/v4.0_4.0.0.0__b77a5c561934e089/System.Core.dll
----------------------------------------
System.Xml

**Comment by seblindfors** [comment]
Download and run the .NET repair tool just to rule out any .NET issues.
https://www.microsoft.com/en-us/download/details.aspx?id=30135

**Comment by topher-au** [comment]
Greetings!

It's likely your plugin DLL files are being blocked by Windows - you need to go to the "plugins" folder wherever you installed WoWmapper ('%LocalAppData%\WoWmapper\plugins' is default) and right click the input dll files located there and select "unblock" from the Properties window.

**Comment by topher-au** [comment]
Greetings! As I have not heard back from you I am going to close this issue.  Please open another issue if you continue to have problems.
